backend/api/Areas/Project/Mapping/Dispose/ProjectMap.cs

```diff
@@ -57,6 +58,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.OfferAmount, src => src.OfferAmount)
                 .Map(dest => dest.Tasks, src => src.Tasks)
                 .Map(dest => dest.ProjectAgencyResponses, src => src.Responses)
+                .BeforeMapping((src, dest) => JsonConvert.PopulateObject(src.Metadata ?? "{}", src)) //use the metadata object to populate fields not stored in DB.
```

Comment:
This allows the mapping to behave the same way it did before these fields were removed, all fields will be mapped directly from the entity to the model since the entity will be populated with the metadata before the mapping occurs.

Note: populateObject respects the field annotations in the Project object, so only [JsonProperty] fields will be populated.